### PR TITLE
Adds Trunk blocks and prefabs

### PR DESCRIPTION
**Pull request by Qwertygiy**
Textures are in [CoreAssets' develop branch.](https://github.com/Terasology/CoreAssets/commit/8e8e6dd248963994adc56549f86d2c0af3e31f4d) Block logic is forthcoming in a pull request to Core.

### CODE REVIEW

File: assets/blocks/TrunkLeft.block [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
+ * A chest is interesting for two reasons: It is asymmetrical (front texture differs) and should store stuff
```
Comment (skaldarnar) [inline assets/blocks/TrunkLeft.block]:
```suggestion
 * A trunk is interesting for two reasons: they are two blocks wide and are asymmetrical (front texture differs from the sides) 
```

File: assets/blocks/TrunkRight.block [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
+ * A chest is interesting for two reasons: It is asymmetrical (front texture differs) and should store stuff
```
Comment (skaldarnar) [inline assets/blocks/TrunkRight.block]:
```suggestion
 * A trunk is interesting for two reasons: they are two blocks wide and are asymmetrical (front texture differs from the sides) 
```

File: assets/prefabs/trunkRegion.prefab [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    "DisplayName": {
+        "name": "Trunk"
+    },
+    "Trunk": {
+        "leftBlockFamily": "CoreBlocks:TrunkLeft",
+        "rightBlockFamily": "CoreBlocks:TrunkRight",
+        "openSound": "CoreAssets:DoorOpen",
+        "closeSound": "CoreAssets:DoorClose"
+    },
+    "Health": {
```
Comment (skaldarnar) [inline assets/prefabs/trunkRegion.prefab:5]: Why are components like `Health` and `Trunk` in both `trunk.prefab` and `trunkRegion.prefab`?

File: assets/prefabs/trunkRegion.prefab [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            0,
+            0,
+            0,
+            0,
+            0
+        ]
+    },
+    "RetainBlockInventory": {},
+    "PlaySoundAction": {
+        "sounds": "engine:click"
```
Comment (skaldarnar) [inline assets/prefabs/trunkRegion.prefab:105]: Why this sound action? I though opening and closing were their own sounds? Or is this about the inventory interaction?

File: assets/prefabs/trunkRegion.prefab (hunk 3 above)
Comment (Qwertygiy) [reply assets/prefabs/trunkRegion.prefab:5]: There was a lot of experimentation going on while trying to figure out which pieces of door and which pieces of chest needed to be combined and where. Health should only be required in the region, and Trunk should only be required in the item.

File: assets/prefabs/trunkRegion.prefab (hunk 4 above)
Comment (Qwertygiy) [reply assets/prefabs/trunkRegion.prefab:105]: I copied it from the chest item; we just don't have separate sounds for that yet.

**Review by skaldarnar — APPROVED** [review]

**Pull request merged**